src/app/components/Table/Table.tsx

```diff
@@ -79,8 +85,8 @@ export const Table: FunctionComponent<TableProps> = ({
         </Tr>
       </Thead>
       <Tbody>
-        {transformRows(rows, columns).map((row, rowIndex) => (
-          <Tr key={row?.originalData?.id ?? rowIndex}>
+        {transformRows(rows, columns).map((row: TableRow, rowIndex) => (
```

Comment:
Why do we not need "typing" on `rowIndex`?

Reply:
Because from `.map` function definition you already know that the second argument is always a number, so the compiler is able to infer it and it is not necessary to declare it.
However, if you prefer I am ok for adding it explicit.